extensions/kubectl-cli/src/extension.ts

```diff
@@ -337,7 +337,44 @@ async function postActivate(
   let releaseToInstall: KubectlGithubReleaseArtifactMetadata | undefined;
   let releaseVersionToInstall: string | undefined;
   let currentVersion = kubectl.version;
-
+  // check if there is a new version to be installed and register the updater
+  let releaseToUpdateTo: KubectlGithubReleaseArtifactMetadata | undefined;
+  let releaseVersionToUpdateTo: string | undefined;
+  const latestAsset = await kubectlDownload.getLatestVersionAsset();
+  const update = {
```

Comment:
Have to move to separate object to be able to update version property, to control card showing ether `Update` or `Upgrade/Downgrade` commands.